tests/e2e/install_suite_test.go

```diff
@@ -21,60 +26,199 @@ var _ = BeforeSuite(func() {
 
 	vel = &veleroCustomResource{
```

Comment:
This is a poorly named struct var to setup the testing context. I would like to rework this slightly.